Add EnvironmentOverrides opening tag

p1p3 opened the pull request:
The example is not compliant with the  http://schemas.microsoft.com/2011/01/fabric schema due the missing EnvironmentOverrides opening tag.

azurecla commented:
Hi __@stevehoberecht__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

p1p3 committed:
Add EnvironmentOverrides opening tag

The example is not compliant with the  http://schemas.microsoft.com/2011/01/fabric schema due the missing EnvironmentOverrides opening tag.
